src/test/java/com/google/javascript/clutz/goog_module_class.js

```diff
@@ -1,6 +1,11 @@
 goog.module('module.Foo');
 
 class A {
+  /**
+   * @param {string} x
+   * @param {...string} rest
+   */
+  constructor(x, ...rest) {}
 }
```

Comment:
I think our test runs here are not properly testing what John is working towards internally - not transpiling to ES5 internally. The question is there a option that we can use to test that, or it is still work in progress, thus not available to upstream maven builds.

Reply:
Looking at the internal change, it looks like there is no way to test the proposed change through the public api, because it requires touching - https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/TranspilationPasses.java#L111

I guess we have a bit of a deadlock, so we should just land this untested, so that john can land his change and which point this test will become the correct one.